src/constants/rdf_predicates.js

```diff
@@ -24,7 +24,10 @@ const RDF_PREDICATES = {
   sender: 'https://schema.org/sender',
   profileName: 'http://xmlns.com/foaf/0.1/name',
   sha256: 'https://schema.org/sha256',
-  accountablePerson: 'https://schema.org/accountablePerson'
+  accountablePerson: 'https://schema.org/accountablePerson',
+  profileImg: FOAF.img,
+  nickname: FOAF.nick,
+  type: RDF.type
```

Comment:
Is this `type` parameter still being used? If not we can remove it.

Reply:
Yeah, think I can remove this.